tests/test_snipping.py

```diff
@@ -487,3 +487,141 @@ def test_cooler_snipper_with_view(request):
         matrix, "foo", "foo", (110_000_000, 120_000_000, 110_000_000, 120_000_000)
     )
     assert snippet.shape is not None
+
+def test_pooled_pileup(request):
```

Comment:
I wonder if we could just add 
```
stack_pooled2 = cooltools.api.snipping.pileup(clr, windows, view_df=None, flank=None, nproc=2)
assert np.array_equal(stack, stack_pooled2, equal_nan=True)
```

to the earlier test, rather than needing to recalculate all of this in the test_pooled_pileup

Reply:
That makes sense. I will only keep one